genesyscloud/outbound/resource_genesyscloud_outbound_campaign_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"terraform-provider-genesyscloud/genesyscloud/outbound_callabletimeset"
```

Comment:
Probably better to define a camelcase variable for this, like how it is done in genesyscloud/outbound/resource_genesyscloud_outbound_init_test.go

Snake case isn't really meant to be in go code